relay-protocol/src/condition.rs

```diff
@@ -106,12 +106,12 @@ macro_rules! impl_cmp_condition {
             /// Path of the field that should match the value.
             pub name: String,
             /// The numeric value to check against.
-            pub value: Number,
+            pub value: Value,
```

Comment:
I didn't wanna add this since the semantics of a comparison with an array are questionable. For `eq`, we defined it as "one of", but for comparisons this seemed weird to me. Thoughts?

Reply:
I personally found it already surprising for `Eq`, usually this would be a `In` operator. So for me then I assumed the value can always be an array and it turns into something like `value <OPERATOR> ANY(value)`. 

I agree though this is weird but we already have a precedent for the array. Though I think we just leave it as is and implement it when we need it (we can bump versions now, yay)